services/state/schedule/schedule.proto

```diff
@@ -100,7 +100,7 @@ message Schedule {
      * The requested expiration time of the schedule as provided by the user.
      * The actual calculated expiration time may be "earlier" than this, but will not be later.
      */
-    Timestamp expiration_time_provided = 10;
+    int64 provided_expiration_seconds = 10;
```

Comment:
This should be singular, it is not a duration, it is a point in time.

Reply:
Fixed.

Reply:
Fixed.